packages/pfff/pfff.0.37.6/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Tools and APIs for program analysis, code visualization, refactoring"
+description: """
+pfff is a set of tools and APIs to perform some static analysis,
+dynamic analysis, code visualizations, code navigations, or
+style-preserving source-to-source transformations such as refactorings
+on source code. For now the effort is focused on PHP but there is also
+good support for C, C++, Java, HTML, Javascript, Css, and preliminary
+support for Erlang, Python, C#, Lisp, Scheme, Haskell, OPA, Sql, and
+even TeX. There is also very good support for OCaml and noweb
+(literate programming) so that pfff can be used on the code of pfff
+itself.
+
+For more information see https://github.com/facebook/pfff/wiki/Main"""
+
+maintainer: "yoann.padioleau@gmail.com"
+authors: [ "Yoann Padioleau <yoann.padioleau@gmail.com>" ]
+homepage: "https://github.com/facebook/pfff/wiki/Main"
+bug-reports: "https://github.com/aryx/pfff/issues"
+dev-repo: "git+https://github.com/aryx/pfff.git"
+
+build: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "depend"]
+  [make]
+  [make "opt"]
+]
+install: [
+  [make "install-libs"]
+]
+remove: [
```

Comment:
The `remove` field is not necessary since opam 2.0. Could you remove it?

Reply:
actually @avsm told me to add it (it was breaking ci/3 compiler tests in my previous PR)

Reply:
This is due to a bug in Docker see here for the opam issue: https://github.com/ocaml/opam/issues/3997

I'm not sure how to fix the isuse in the meantime though